cocos/ui/editbox/edit-box.ts

```diff
@@ -759,7 +759,6 @@ export class EditBox extends Component {
 
         if (placeholderLabel) {
             placeholderLabel.node._uiProps.uiTransformComp!.setContentSize(size.width - LEFT_PADDING, size.height);
-            placeholderLabel.lineHeight = size.height;
```

Comment:
It should assign from label's lineHeight attribute from editor directly like textLabel. No need to assign value.

Reply:
If it is not set in editor. Will it also work correctly?

Reply:
> If it is not set in editor. Will it also work correctly?

Sure. label component has initial value when created.

Reply:
> It should assign from label's lineHeight attribute from editor directly like textLabel. No need to assign value here.

And there's no meaningful to assign placeholderLabel.lineHeight = size.height;